networkx/conftest.py

```diff
@@ -210,6 +210,7 @@ def add_nx(doctest_namespace):
     "linalg/laplacianmatrix.py",
     "utils/misc.py",
     "algorithms/centrality/laplacian.py",
+    "algorithms/distance_measures/kemeny_constant.py",
```

Comment:
This looks like it might be left over from an earlier version of this PR? AFAICT there is no file by this name.

Having noticed this, I was surprised that the CI was passing but it turns out we don't run doctests without default dependencies. I don't remember if this is intentional or not :upside_down_face: .

Either way - I propose to just remove these two added lines. Let's leave worrying about testing configurations to a different PR.